operators/src/source/ogr_source/mod.rs

```diff
@@ -610,6 +610,64 @@ impl FeaturesProvider<'_> {
         Ok(())
     }
 
+    ///  Creates a time filter string.
+    fn create_time_filter_string(
+        time_type: OgrSourceDatasetTimeType,
+        mut time_interval: TimeInterval,
+        driver: &str,
+    ) -> Option<String> {
+        match driver {
+            "PostgreSQL" => {
+                let postgres_range: TimeInterval = // https://www.postgresql.org/docs/current/datatype-datetime.html, capped by max supported TimeInstance value
+                    TimeInterval::new_unchecked(-210_895_056_000_000, 8_210_266_876_799_999);
+
+                if !postgres_range.contains(&time_interval) {
+                    return None;
```

Comment:
shouldn't that be an error at least log a warning?

Reply:
I can log it as a warning if you want, something like "Time filter was not pushed down to PG because it exceeds the possible range". However, I don't consider it to be an error. What it means is simply that we cannot safely create a WHERE clause which Postgres can use to filter by time on its end, because the endpoints of the query time range are out of PG's timestamp type range. So in this case we skip the filtering and filter on the GE side. In practice, if the column is in fact of the timestamp type, we won't get any values out of range so we could also clip to PG's supported range. But then it would not filter anything anyways so we can just omit the filter as well, which I consider to be the safer approach.
(If the column were of type string, for example, PG would not try to convert the endpoints to a PG timestamp, so then it would not fail. But comparing timestamps by string comparison leads to a whole other lot of potential issues, so I don't think this should be addressed in this PR. The push down is only used for GBIF rn and there we know that the timestamps are represented by the PG timestamp type.)